config/jobs/kubernetes/kops/kops-config.yaml

```diff
@@ -199,7 +199,7 @@ presubmits:
         - --aws-cluster-domain=test-cncf-aws.k8s.io
         - --check-leaked-resources=false
         - --cluster=
-        - --extract=ci/k8s-stable3
+        - --extract=ci/k8s-stable2
```

Comment:
same here, but `stable3` in name vs. `stable2` in use

Reply:
job removed